core/nandtool/nandtool.c

```diff
@@ -112,13 +114,23 @@ static int nandtool_flash(const char *path, unsigned int start, int raw)
 			}
 
 			offs += boffs;
-			printf("\rFlashing %s %u%%...", path, (unsigned int)(100 * offs / stat.st_size));
+
+			perc = 100 * offs / stat.st_size;
```

Comment:
Changes nothing, but I like to explicitly force operation order. Just a suggestion 
```suggestion
			perc = (100 * offs) / stat.st_size;
```

Reply:
added to please You :rofl: